src/internal/methods/NeoEspBitBangMethod.h

```diff
@@ -180,7 +181,8 @@ template<typename T_SPEED, typename T_INVERTED> class NeoEspBitBangEncode : publ
             T_SPEED::Period,
             sizePixel,
             T_SPEED::TInterPixel,
-            T_INVERTED::IdleLevel);
+            T_INVERTED::IdleLevel,
```

Comment:
Some LEDs can't have the timing between pixels extended beyond the normal variance (~150us).  So this doesn't work for all LEDs.